samples/sampler/stories/qa/VideoPlayer.js

```diff
@@ -2,6 +2,7 @@ import Button from '@enact/sandstone/Button';
 import {MediaControls} from '@enact/sandstone/MediaPlayer';
 import VideoPlayer, {Video} from '@enact/sandstone/VideoPlayer';
 import {select} from '@enact/storybook-utils/addons/controls';
+import Popup from '@enact/sandstone/Popup';
```

Comment:
We usually sort import statements alphabetically.

Reply:
Thank you. I fixed.